lib/models/IonUpdateRequestModel.ts

```diff
@@ -0,0 +1,22 @@
+import IonPublicKeyModel from './IonPublicKeyModel';
+import IonServiceModel from './IonServiceModel';
+import OperationType from '../enums/OperationType';
+
+/**
+ * Data model representing a public key in the DID Document.
+ */
+export default interface IonUpdateRequestModel {
+  type: OperationType;
+  didSuffix: string;
+  revealValue: string;
+  delta: {
+    updateCommitment: string,
+    patches: {
+        action: string,
+        servicesToAdd?: IonServiceModel[],
```

Comment:
Sanity: `servicesToAdd` seems wrong?

Also, maybe a clearer (but more tedious) approach would be something like:
`patches: (IonAddServicesActionModel | IonRemoveServicesActionModel | IonAddPublicKeysActionModel | IonRemovePublicKeysActionModel )[]`